Tools/clinic/libclinic/__init__.py

```diff
@@ -0,0 +1,48 @@
+from typing import Final
+
+from .accumulators import (
+    _TextAccumulator,
+    _text_accumulator,
+    text_accumulator,
+)
+from .formatting import (
+    c_repr,
+    docstring_for_c_string,
+    indent_all_lines,
+    pprint_words,
+    suffix_all_lines,
+    wrapped_c_string_literal,
+)
+
+
+CLINIC_PREFIX: Final[str] = "__clinic_"
+CLINIC_PREFIXED_ARGS: Final[set[str]] = {
+    "_keywords",
+    "_parser",
+    "args",
+    "argsbuf",
+    "fastargs",
+    "kwargs",
+    "kwnames",
+    "nargs",
+    "noptargs",
+    "return_value",
+}
```

Comment:
(Just annotating with `Final` here is enough; since it's a global constant, mypy can easily infer the "rest of the type" here)

```suggestion
CLINIC_PREFIXED_ARGS: Final = frozenset({
    "_keywords",
    "_parser",
    "args",
    "argsbuf",
    "fastargs",
    "kwargs",
    "kwnames",
    "nargs",
    "noptargs",
    "return_value",
})
```